Source/ACE.Database/CharacterDatabase.cs

```diff
@@ -260,12 +314,12 @@ public bool IsNameAvailable(string name)
                 c.TemplateOption = result.Read<uint>(0, "templateOption");
                 c.StartArea = result.Read<uint>(0, "startArea");
 
-                c.Position = await this.GetPosition(guid);
+                c.PhysicalPosition = await this.GetPosition(guid);
```

Comment:
Change PhysicalPosition to Location?

Character.Location = current position right?